e2e/scripts/st_plotly_chart.py

```diff
@@ -240,6 +240,7 @@
     ),
 )
 fig.update_layout(height=300, width=600)
+fig.update_layout(title_font_size=30)
```

Comment:
The linked issue also applies some other styling changes: 

```
fig.update_layout(
    font_family="Courier New",
    font_color="blue",
    title_font_family="Times New Roman",
    title_font_color="red",
    legend_title_font_color="green"
    title_font_size=40,
)
```

Maybe we add some more layout options to our e2e test as well